### DISCUSSION TITLE
%E5%9F%BA%E7%A1%80%E6%B7%B1%E5%85%A5/%E8%AF%BE%E7%A8%8B%E6%8E%A8%E8%8D%90/%E6%95%B0%E5%AD%A6%E8%BF%9B%E9%98%B6/Probability/

### DISCUSSION THREAD

**Discussion by giscus** [MingLLuo/CS-Plan#14] [Announcements]
# %E5%9F%BA%E7%A1%80%E6%B7%B1%E5%85%A5/%E8%AF%BE%E7%A8%8B%E6%8E%A8%E8%8D%90/%E6%95%B0%E5%AD%A6%E8%BF%9B%E9%98%B6/Probability/

https://cs-plan.com/%E5%9F%BA%E7%A1%80%E6%B7%B1%E5%85%A5/%E8%AF%BE%E7%A8%8B%E6%8E%A8%E8%8D%90/%E6%95%B0%E5%AD%A6%E8%BF%9B%E9%98%B6/Probability/

**Comment by yourauroral** [comment]
我做的是华章出版的概率与计算，书后题的答案可以在chegg上有所查找，但有所错误，总之，建议没有一定水平的作者书后题挑着做，难度比较大。
